source/read/count.txt

```diff
@@ -0,0 +1,138 @@
+.. _kotlin-sync-count:
+
+===============
+Count Documents
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation, code example
+
+Overview
+---------
+
+In this guide, you can learn how to retrieve accurate and estimated counts of the
+number of documents in a collection.
+
+.. _kotlin-sync-accurate-count:
+
```

Comment:
S: not necessary, but it might be helpful to use one of the Atlas sample datasets for the code examples on this page and introduce it here (similar to the [C++ count documents](https://preview-mongodbnorareidy.gatsbyjs.io/cpp-driver/DOCSP-40801-count/read/count/) page). This could help make the current code examples a little more specific and would provide an output